public/app/core/components/AppChrome/AppChrome.tsx

```diff
@@ -280,9 +279,10 @@ const getStyles = (theme: GrafanaTheme2, headerHeight: number) => {
       // the `Resizeable` component overrides the needed `position` and `height`
       // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
       position: 'fixed !important' as 'fixed',
+      // When two bars, position from first level (next to Share button); otherwise from full header
```

Comment:
```suggestion
      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
      position: 'fixed !important' as 'fixed',
```

This comment does not seem relevant anymore